pkg/codegen/go/gen_program_expressions.go

```diff
@@ -1078,7 +1086,15 @@ func (g *generator) genApply(w io.Writer, expr *model.FunctionCallExpression) {
 	applyArgs, then := pcl.ParseApplyCall(expr)
 	isInput := false
 	retType := g.argumentTypeName(then.Signature.ReturnType, isInput)
-	// TODO account for outputs in other namespaces like aws
+
+	// Add all argument/return types to imports.
+	for _, applyArg := range applyArgs {
+		schemaType, _ := pcl.GetSchemaForType(applyArg.Type())
```

Comment:
I believe not all `applyArgs` will necessarily have a schema type, lets only add type imports if they actually have a schema type:
```go
if schemaType, ok := pcl.GetSchemaForType(applyArg.Type()); ok {
    g.collectTypeImports(g.program, schemaType)
}
```